src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -65,6 +65,9 @@
     @Argument(fullName = StandardArgumentDefinitions.VERBOSITY_NAME, shortName = StandardArgumentDefinitions.VERBOSITY_NAME, doc = "Control verbosity of logging.", common = true, optional = true)
     public Log.LogLevel VERBOSITY = Log.LogLevel.INFO;
 
+    @Argument(fullName = "log_to_file", shortName = "log", doc = "Set the logging location", common = true, optional = true)
```

Comment:
I was using the GATK3 names. The other argument names in the file use the same convention.